buildenv

```diff
@@ -0,0 +1,37 @@
+export ZOPEN_TARBALL_URL="https://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.10.2.tar.bz2"
+export ZOPEN_TARBALL_DEPS="make bzip2 tar curl perl automake autoconf m4 sed gettext zoslib diffutils libgpg"
+export ZOPEN_TYPE="TARBALL"
+export ZOPEN_EXTRA_LDFLAGS="${ZOPEN_EXTRA_LDFLAGS} -L\$LIBGPG_HOME/lib"
```

Comment:
```suggestion
export ZOPEN_EXTRA_LDFLAGS="${ZOPEN_EXTRA_LDFLAGS} -L\$LIBGPGERROR_HOME/lib"
```
Should it be LIBGPGERROR_HOME with the name change or is it still LIBGPG_HOME?

Reply:
I have corrected this, thank you!